Library/Homebrew/test/cmd/bundle_spec.rb

```diff
@@ -1,23 +1,25 @@
 # frozen_string_literal: true
 
-RSpec.describe "brew bundle", :integration_test do
-  describe "check" do
-    it "checks if a Brewfile's dependencies are satisfied", :needs_network do
-      setup_remote_tap "homebrew/bundle"
+require "cmd/shared_examples/args_parse"
 
-      HOMEBREW_REPOSITORY.cd do
-        system "git", "init"
-        system "git", "commit", "--allow-empty", "-m", "This is a test commit"
-      end
+RSpec.describe "Homebrew::Cmd::BundleCmd", :integration_test, :needs_network do
+  before { setup_remote_tap "homebrew/bundle" }
+
+  it_behaves_like "parseable arguments", command_name: "bundle"
```

Comment:
Explicitly providing `command_name` here is suboptimal, but we don't know the command name until we've loaded the class. However, for remote taps, we don't know the path to the file the class is defined in until we've invoked `Commands.external_ruby_v2_cmd_path` with the command name 🔁